libopenage/renderer/camera/camera.h

```diff
@@ -23,6 +25,13 @@ class UniformBuffer;
 
 namespace camera {
 
+/**
+ * Defines constant boundaries for the camera's view in the X and Z axes.
+ */
+struct CameraBoundaries {
+	float x_min, x_max, z_min, z_max;
```

Comment:
You should document these members. I also prefer them being written out as individual declerations because that is easier to read sometimes, e.g.

```cpp
struct CameraBoundaries {
	float x_min;
	float x_max;
	float z_min;
	float z_max;
};
```